plugins/docker/default.yml

```diff
@@ -16,10 +16,12 @@ endpoint:
 # plugins
 plugin:
   - basicmessage_storage.v1_0
+  - connections.v1_0
```

Comment:
I think the connections plugin is just `connections`?

Reply:
I am confused about this the docs say it is just `connections`, however the plugin structure seems to indicate (as far as I can tell) the `v1_0` path/subfolder in the naming might be required?

Reply:
Ya. This was a bit confusing when we did it. The protocol should load in acapy because of the v1_0 directory but it didn't work correctly for some reason that isn't clear.

Reply:
Even using `conenctions` I don't see the right set of endpoints in swagger... 🤔

Reply:
There was a typo in the values file, should be fixed now... 😅